src/serverlist.cpp

```diff
@@ -443,6 +455,18 @@ void CServerListManager::CentralServerQueryServerList ( const CHostAddress& Inet
                 {
                     vecServerInfo[iIdx].HostAddr = ServerList[iIdx].LHostAddr;
                 }
+                else if ( !NetworkUtil::IsPrivateNetworkIP(InetAddr.InetAddr) && NetworkUtil::IsPrivateNetworkIP(vecServerInfo[iIdx].HostAddr.InetAddr) && !NetworkUtil::IsPrivateNetworkIP(ServerList[iIdx].LHostAddr.InetAddr) )
```

Comment:
This condition is really long. If you have any ideas how to simplify / change formatting, let me know. There seem to be other instances of such long conditions though.

Reply:
Break it into multiple lines by adding a newline after each `&&` (see `protocol.cpp` for examples of this).

Reply:
Thanks for the `protocol.cpp` pointer. I have adjusted this part to match the coding style there.